src/components/App/StepComponentMap.js

```diff
@@ -33,4 +33,24 @@ const shallowFlattenList = list => [].concat(...list);
 
 const stepsToComponents = (stepDefaultOptions, steps) => shallowFlattenList(steps.map( step => stepToFormatToComponents(stepDefaultOptions, step)));
 
+class DocumentCapture extends Component {
+  render (options) {
+    return <Capture method='document' autoCapture={true} {...options} />
+  }
+}
+
+DocumentCapture.defaultProps = {
+  useWebcam: false
+}
+
+class FaceCapture extends Component {
```

Comment:
you can turn this into stateless components, like this:
```
const FaceCapture = options =>
  <Capture method='face' autoCapture={false} {...options} />
```

You can still set `defaultProps` in the same way.